Tracking/TrackLength/README.md

```diff
@@ -0,0 +1,126 @@
+## Description
+
+TrackLength is the Marlin processor that calculates track length and square root of the harmonic mean of the squared momentum of charged particles.
```

Comment:
Maybe add one sentence here, that says that it was previously a part of TOFEstimators.